lib/network_config.ml

```diff
@@ -76,20 +76,25 @@ let read_management_conf () =
 				ports = [device, {default_port with interfaces = [device]}];
 				persistent_b = true
 				} in
-			match vlan with
-			| None ->
+			if bridge_name = "" then
 				[device, phy_interface; bridge_name, bridge_interface],
```

Comment:
These two lines are the same as the lines below in the `None` case. That could be deduplicated.
But more importantly, `bridge_name` is `""` here, so why is it used at all? Shouldn't we just leave the entire config and return `[], []`?